README.md

```diff
@@ -10,15 +10,40 @@
 ![GitHub last commit](https://img.shields.io/github/last-commit/banurapp/interface-web)
 
 
+## What is this for?
+
+This is the main interface that connects to each Banur community—the client that brings it all together.
 Welcome to the web interface for the Banur Project! We're so excited to have you.
 
 ## 🛠 Contributing
 
-Please read the [Contributions](CONTRIBUTING.md) file to get started.
+Before getting started at all, please read our [Code Of Conduct](CODE_OF_CONDUCT.md), then learn how to [Contribute](CONTRIBUTING.md).
 
-## What is this for?
+## ✍️ Getting Started
+
+### Clone
+
+```bash
+git clone https://github.com/banurapp/web.git
+```
+
+### Install dependencies
+
+```bash
+npm install
```

Comment:
We are using yarn. Can you change these command to use yarn instead of npm?

Reply:
I'll add the yearn. But is it ok if i also keep npm commands?

Reply:
also, i don't know what the yarn commands are since I don't really use it

Reply:
Sorry but we have chosen npm for this project. It is kinda one way or the other with this kinda thing.

Reply:
yeah i added yarn, but can i keep the npm commands?

Reply:
No, if some uses npm then it will generate a pack-lock.json instead of the yarn.lock that we are using. It is kinda one way or the other with this kinda thing.